src/studio/nuxeo-studio/working-in-studio/migration-services.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration Services
+review:
+    comment: ''
+    date: '2018-12-05'
+    status: ok
+tree_item_index: 950
+toc: true
+---
+The JSF UI to Web UI migration tool is a safe and automated way to create, within a few minutes, an equivalent Web UI configuration based on your existing JSF UI configuration.
```

Comment:
I would remove "within a few minutes" as it is our first iteration, and we have no idea on the execution time with complex doc types